praetorian_cli/sdk/entities/agents.py

```
@@ -36,6 +39,97 @@ def affiliation_filename(self, agent_type: str, key: str) -> str:
+        self._conversation_id = conversation_id
+
+        # Snapshot existing messages so we only process new ones
+        last_key = ''
+        try:
+            existing, _ = self.api.search.by_key_prefix(
+                f'#message#{conversation_id}#', user=True
+            )
+            if existing:
+                last_key = max(m.get('key', '') for m in existing)
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)